crates/ruff/src/lib.rs

```diff
@@ -287,6 +287,9 @@ pub fn check(args: CheckCommand, global_options: GlobalConfigArgs) -> Result<Exi
     if show_fixes {
         printer_flags |= PrinterFlags::SHOW_FIX_SUMMARY;
     }
+    if output_format == OutputFormat::Full {
```

Comment:
Should this be gated behind preview mode to get some user feedback before enabling it for everyone?

Reply:
I don't think it's strictly necessary, but we may want to do so if it's going to change significantly (i.e. by being split into follow-ups)